packages/react-components/src/components/InputBoxButton.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ICalloutContentStyles, IconButton, Stack, TooltipHost, mergeStyles, useTheme } from '@fluentui/react';
+import { useState } from 'react';
+import { isDarkThemed } from '../theming/themeUtils';
+import React from 'react';
+import { iconWrapperStyle, inputBoxButtonStyle, inputBoxButtonTooltipStyle } from './styles/InputBoxButton.style';
+
+/**
+ * Props for displaying a send button besides the text input area.
+ *
+ * @private
+ */
+export type InputBoxButtonProps = {
+  onRenderIcon: (isHover: boolean) => JSX.Element;
+  onClick: (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+  className?: string;
+  id?: string;
```

Comment:
Not really a part of the changes but I'm wondering how `id` prop is used here and if it's really needed.
As per documentation, id should be unique and with message edit functionality, the edit box can be shown for a few messages at the same moment of time

Reply:
That's a good question. Let's discuss this at the next sync.